app/rtc_compare.py

```diff
@@ -743,7 +768,8 @@ def _print_first_value_diff(image_1, image_2, prefix):
                   f' (x: {j}, y: {i})'
                   f' whereas input 2 has value "{image_2[i, j]}"'
                   ' in the same position. '
-                  f' Difference: {image_2[i, j] - image_1[i, j]}')
+                  f' Difference: {image_2[i, j] - image_1[i, j]}, ',
+                  f' Ratio of Failed pixel : {failed_pixel_ratio:e}')
```

Comment:
```suggestion
                  f' Difference: {image_2[i, j] - image_1[i, j]}')
```

Reply:
Let's print this in the caller code so we don't have to recompute `failed_pixel_ratio` again.

Reply:
`i` and `j` are computed in `_print_first_value_diff()`, so I will let the code to print the percentage of the failure only when the ratio is above `RTC_S1_PRODUCTS_FAILED_PIXEL_RATIO_TOLERANCE`.
Also Please note that I've put early return here.